datavalid.yml

```diff
@@ -60,6 +60,10 @@ schemas:
           - female
           - male
       - name: agency
+      - name: class_id
+      - name: assignment_id 
```

Comment:
Would be nice to add `description` to more obscured name like `class_id` and `assignment_id`.

Reply:
Agreed. I have no idea what they actually mean, yet. Working on getting more information.

Reply:
I think we should always discard columns that we don't understand. Data without meaning cannot serve any purpose. If the intention is to remind ourselves to find out more about some columns, then I think it's better to leave `# TODO:` comments in the clean scripts. Unless you have other reasons to leave them in.

Reply:
My original thinking was that someone who is downloading these tables for research purposes may know what they mean, as they are likely to have more insight than us into NOPD. Happy to leave `# TODO:` comments in the clean scripts if you think that concern is unwarranted.

Reply:
Yeah, it's just hard for me to imagine a case where someone who look at the data we produce and would know more than us. They will probably have to look at our scripts anyway and even the source data as well. And the web developers will have to modify their SQL schema to accommodate any new field. So it's probably more trouble than it's worth.

Reply:
Makes sense to me. Removed the columns. This is ready for review.